src/components/password-input/password-input.tsx

```diff
@@ -34,7 +37,12 @@ const PasswordInput = React.forwardRef<
     const [showPassword, setShowPassword] = useState(false);
     const [capsError, setCapsError] = useState(false);
 
-    const { handlePasswordValidation, passwordState } = usePasswordSchema();
+    const { handlePasswordValidation, passwordState, isValidPassword } =
+      usePasswordSchema();
+
+    useEffect(() => {
+      onPasswordValidation?.(isValidPassword);
+    }, [isValidPassword, onPasswordValidation]);
```

Comment:
@Sayemhaque  @soyeba3  this is how we can use callback to pass value to the consumer